receiver/prometheusreceiver/metrics_receiver_test.go

```diff
@@ -263,12 +270,17 @@ func verifyTarget1(t *testing.T, td *testData, mds []internaldata.MetricsData) {
 			Name:        "go_threads",
 			Description: "Number of OS threads created",
 			Type:        metricspb.MetricDescriptor_GAUGE_DOUBLE,
+			LabelKeys:   []*metricspb.LabelKey{{Key: "instance"}, {Key: "job"}},
```

Comment:
Should `instance` and `job` be part of the resource instead of labels?

Reply:
You are correct, working on it and will send another version.